msf_updates/src/pose_msf/pose_sensormanager.h

```diff
@@ -170,16 +170,16 @@ class PoseSensorManager : public msf_core::MSF_SensorManagerROS<
     P.setZero();  // Error state covariance; if zero, a default initialization in msf_core is used
 
     p_vc = pose_handler_->GetPositionMeasurement();
-    q_cv = pose_handler_->GetAttitudeMeasurement();
+    q_vc = pose_handler_->GetAttitudeMeasurement();
```

Comment:
Depending on the subscribed topic (ptam/vicon) the pose provided can be
either q_cv (ptam) or q_vc/q_wc (vicon). If you look at the implementation
of the pose update you see that there is a fix-parameter (from yaml) that
defines if the pose measurement needs to be inverted or not before being
applied as update. You would need to add the same logic here.

On Tue, Jun 16, 2015, 09:39 Tim Oberhauser <notifications@github.com> wrote:

> In msf_updates/src/pose_msf/pose_sensormanager.h
> <https://github.com/ethz-asl/ethzasl_msf/pull/116#discussion_r32497243>:
>
> > @@ -170,16 +170,16 @@ class PoseSensorManager : public msf_core::MSF_SensorManagerROS<
> >      P.setZero();  // Error state covariance; if zero, a default initialization in msf_core is used
> >
> >      p_vc = pose_handler_->GetPositionMeasurement();
> > -    q_cv = pose_handler_->GetAttitudeMeasurement();
> > +    q_vc = pose_handler_->GetAttitudeMeasurement();
>
> Sorry about that! For me this was the case, so this notation made it
> easier to understand for me... Simon, can you please elaborate on your idea?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethz-asl/ethzasl_msf/pull/116/files#r32497243>.
>